server/app/controllers/user_infos_controller.rb

```diff
@@ -6,25 +6,31 @@ def show
     if User.exists?(params[:user_id])
       user = User.find(params[:user_id])
       show_user_info = user.user_info
-      render json: show_user_info, status: :ok, serializer: UserInfosSerializer
+      if show_user_info == nil
+        render json: { message: "Info doesn't exist." }, status: 404
+        # render json: show_user_info.errors.full_messages, status: 404
+      else
+        render json: show_user_info, status: :ok, serializer: UserInfosSerializer
+      end
     else
-      render json: { message: "Info doesn't exist" }, status: 404
+      render json: { message: "User doesn't exist" }, status: 404
     end
   end
 
-  def create
-    user = current_user
-    user_info = user.build_user_info(user_info_params)
-    if user_info.save
-      render json: user_info, status: :created, serializer: UserInfosSerializer
-    else
-      render json: user_info.errors.full_messages, status: :unprocessable_entity
-    end
-  end
+  # def create
+  #   user = current_user
+  #   user_info = user.build_user_info(user_info_params)
+  #   if user_info.save
+  #     render json: user_info, status: :created, serializer: UserInfosSerializer
+  #   else
+  #     render json: user_info.errors.full_messages, status: :unprocessable_entity
+  #   end
+  # end
 
   def update
     user = current_user
-    user_inf = current_user.user_info
+    user_inf = current_user.user_info || user.build_user_info({})
+    user_inf.save
     if user.id === user_inf.user_id
```

Comment:
[`Integer#===`](https://ruby-doc.org/core-3.1.1/Integer.html#method-i-3D-3D-3D) is an alias for `Integer#==`, so that it is better to use `==`, which is more common in Ruby world